### PR TITLE
Add examples for configs

### PR DESCRIPTION
**Pull request by mistyhacks**
Signed-off-by: Misty Stanley-Jones <misty@docker.com>

Fixes https://github.com/docker/docker.github.io/issues/5878

cc/ @mbentley 

baby starfish

### CODE REVIEW

File: docs/reference/commandline/service_create.md [hunk 1]
```diff
@@ -174,12 +174,40 @@ $ docker service create --name redis \
...
+Secrets are located in `/run/secrets` in the container if no target is specified.
+If no target is specified, the name of the secret is used as the in memory file
+in the container. If a target is specified, that is used as the filename. In the
+example above, two files are created: `/run/secrets/ssh` and
 `/run/secrets/app` for each of the secret targets specified.
 
+### Create a service with configs
+
+Use the `--config` flag to give a container access to a
+[secret](config_create.md).
```
Comment (vdemeester) [inline docs/reference/commandline/service_create.md]: s/secret/config/

Comment (mistyhacks) [reply docs/reference/commandline/service_create.md]: Oops, I knew I would miss one. 😊 Fixed!

File: docs/reference/commandline/service_create.md [hunk 2]
```diff
@@ -174,12 +174,40 @@ $ docker service create --name redis \
...
+### Create a service with configs
+
+Use the `--config` flag to give a container access to a
+[config](config_create.md).
+
+Create a service with a config. The config will be mounted into `redis-config`,
+be owned by the user who runs the command inside the container (often `root`),
+and have file mode `0444` or world-readable. You can specify the `uid` and `gid`
+as numerical IDs or names. The `mode` is specified as a 4-number sequence such
+as `0755`. If a `umask` is set in the container, it impacts the `mode` you set.
```
Comment (thaJeztah) [inline docs/reference/commandline/service_create.md]: > If a `umask` is set in the container, it impacts the `mode` you set.

This was actually a bug, so not sure we should document this (it currently reads as "by design")

File: docs/reference/commandline/service_create.md [hunk 3]
```diff
@@ -174,12 +174,40 @@ $ docker service create --name redis \
...
 
+### Create a service with configs
+
+Use the `--config` flag to give a container access to a
+[config](config_create.md).
+
+Create a service with a config. The config will be mounted into `redis-config`,
+be owned by the user who runs the command inside the container (often `root`),
+and have file mode `0444` or world-readable. You can specify the `uid` and `gid`
+as numerical IDs or names. The `mode` is specified as a 4-number sequence such
```
Comment (thaJeztah) [inline docs/reference/commandline/service_create.md]: Do we need to document that, when using names, those names should be present inside the container?

**Pull request closed without merging**